.ci/openshift-ci/buildall_install.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Build kata-containers and install into a given destination directory
+#
+
+set -e
+
+if [ -z "$1" ];then
+	echo "Usage: $0 path/to/install/dir"
+	exit 1
+fi
+
+export DESTDIR=$1
```

Comment:
I'd consider quoting all variables "just in case" ;)

Reply:
yeah, you never know :)

Reply:
Latest version of the script has all variables quoted.